test/functional/tests.py

```diff
@@ -807,6 +807,68 @@ def test_bitswap_unwant(self):
         result = self.api.bitswap_unwant(key='QmZTR5bcpQD7cFgTorqxZDYaew1Wqgfbd2ud9QqGPAkK2V')
         self.assertTrue(result is not None)
 
+@skipIfOffline()
+class IpfsApiPubSubTest(unittest.TestCase):
+
+    def setUp(self):
+        self.api = ipfsapi.Client()
+
+    def test_pubsub_pub_sub(self):
```

Comment:
Any ideas why this test fails on the CI system? (Error is at https://travis-ci.org/ipfs/py-ipfs-api/jobs/371828905#L621.)

Reply:
Actually the issue may simply be that the versions of `go-ipfs` used by the CI system is too old. Try updating the URL in https://github.com/ipfs/py-ipfs-api/blob/master/.travis.yml#L17 to whatever the URL of the latest Linux-x86_64 version is. The issue should likely be fixed by that.

Reply:
We've definitely seen instances of that in the past, so I agree that this is likely the issue.